pkg/gardenadm/cmd/token/create/create.go

```diff
@@ -53,7 +60,17 @@ gardenadm token create`,
 	return cmd
 }
 
-func run(_ context.Context, opts *Options) error {
-	opts.Log.Info("Not implemented")
-	return nil
+func run(ctx context.Context, opts *Options) error {
+	clientSet, err := tokenutils.CreateClientSet(ctx, opts.Log)
+	if err != nil {
+		return fmt.Errorf("failed creating client set: %w", err)
+	}
+
+	split := strings.Split(opts.Token, ".")
+	if len(split) != 2 {
+		return fmt.Errorf("token must be of the form %q, but got %q", validBootstrapToken, opts.Token)
```

Comment:
This error can never occur if `Options.Validate()` has been called.
Should we store the token as separate ID and secret fields of the `Options` struct instead? You could even use a dedicated `type Token struct { ID, Secret string }`